src/components/shared/SampleTooltipContents.js

```diff
@@ -158,6 +178,8 @@ export class SampleTooltipContents extends React.PureComponent<Props> {
             rangeFilteredThread={rangeFilteredThread}
             categories={categories}
             implementationFilter={implementationFilter}
+            zeroAt={zeroAt}
+            interval={interval}
```

Comment:
you'll be able to remove this after you move the new properties to the right props object.